### DISCUSSION TITLE
Não consigo acessar o container externamente

### DISCUSSION THREAD

**Discussion by joaoggtinoco** [podman-desktop/podman-desktop#4532] [Q&A]
Olá, por algum motivo meu container não está subindo a porta mapeada para que eu acesso o container externamente.
Externamente seria de fora do servidor que hospeda o container.
Ao colocar o container em execução eu verifico se a porta mapeada foi aberta para acesso externo com comando netstat -na | more e percebo que não existe.
Alguém poderia me ajudar ?

**Comment by jeffmaury** [comment]
English translation:

Hello, for some reason my container is not coming up with the mapped port so that I can access the container externally.

Externally would be from outside the server hosting the container.

When I start the container I check if the mapped port has been opened for external access with the netstat -na | more command and I realize that it doesn't exist.

Could someone help me?

**Comment by jeffmaury** [comment]
@joaoggtinoco on which OS/platform ?

**Reply by joaoggtinoco** [reply]
you already helped me on discord, thank you very much
